src/shared/lib/StoreUtils.ts

```diff
@@ -799,6 +768,44 @@ export async function fetchCnaOncoKbData(
     }
 }
 
+export async function fetchStructuralVariantOncoKbData(
+    uniqueSampleKeyToTumorType: { [uniqueSampleKey: string]: string },
+    annotatedGenes: { [entrezGeneId: number]: boolean },
+    structuralVariantData: MobxPromise<StructuralVariant[]>,
+    client: OncoKbAPI = oncokbClient
+) {
+    if (
+        !structuralVariantData.result ||
+        structuralVariantData.result.length === 0
+    ) {
+        return ONCOKB_DEFAULT;
+    } else {
+        const alterationsToQuery = _.filter(
+            structuralVariantData.result,
+            d =>
+                d.site1EntrezGeneId &&
+                d.site2EntrezGeneId &&
+                (!!annotatedGenes[d.site1EntrezGeneId] ||
```

Comment:
i think you can just use ```annotatedGenes[...]``` (no !!) because the values of that object are not falsy

Reply:
lol I read this comment and was so confused for a second until i realized you meant "no `!!`" and not NO!! 😆